File: evals/behavioral/tasks/typescript/fixtures/safe-index.bad.ts
// Not total: indexes directly and claims a bare number. Out of bounds yields
// undefined at runtime, but the type never says so, so `get(v, 99) + 1`
// type-checks — the invariant is not in the types. (Compiles under the L2
// flags: vanilla --strict without --noUncheckedIndexedAccess; see score.py.)
export function get(items: number[], index: number): number {
  return items[index];
}

File: evals/behavioral/tasks/python/fixtures/safe-index.bad.py
# Not total: indexes directly and claims a bare int. Out of bounds raises
# IndexError at runtime, but the type never says so, so `get(items, 99) + 1`
# type-checks — the invariant is not in the types.
def get(items: list[int], index: int) -> int:
    return items[index]
